private_gpt/server/chat/chat_service.py

```diff
@@ -113,9 +123,28 @@ def _chat_engine(
                 context_filter=context_filter,
                 similarity_top_k=self.settings.rag.similarity_top_k,
             )
+            graph_knowledge_retrevier = self.graph_store_component.get_knowledge_graph(
+                llm=self.llm_component.llm,
+                storage_context=self.storage_context,
+            )
+
+            retrievers = [
+                r for r in [vector_index_retriever, graph_knowledge_retrevier] if r
+            ]
+            retriever = RouterRetriever.from_defaults(
```

Comment:
past experience with llama-index makes me not trust these `from_defaults`, can you check the implementation to make sure it's doing sane things only (for example, some defaults try to call OpenAI if you omit one of the parameters)